frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -94,31 +99,44 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
               touch={getIn(formikProps.touched, withNameSpace('address.line1'))}
               displayErrorTooltips
             />
-          </Row>
+          </Form.Row>
         </Col>
         <Col md={1}>
-          <h1>OR</h1>
+          <h5>OR</h5>
         </Col>
-        <Col md={5}>
+        <Col md={5} className="instruction">
           <p>
             Click on the pin, and then click your desired location on the map to pull the parcel
             details.
           </p>
           <Row>
             <Col className="marker-svg">
-              <DraftMarkerButton
+              <SearchMarkerButton
                 onClick={(e: any) => {
                   setMovingPinNameSpace(nameSpace ?? '');
                   e.preventDefault();
                 }}
               >
                 <ParcelDraftIcon className="parcel-icon" />
-              </DraftMarkerButton>
+              </SearchMarkerButton>
             </Col>
           </Row>
         </Col>
       </Row>
-      <Row noGutters>
+      <Row
+        noGutters
+        className={classNames(
+          'section',
+          getIn(formikProps.values, withNameSpace('address.line1')) === '' &&
```

Comment:
disable the information form if none of the search fields have been filled in. I've replaced this check with a check for whether or not the hidden lat/lng fields or filled in my most recent code.